core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -700,6 +758,58 @@ oppia.factory('explorationStatesService', [
         return (
           validatorsService.isValidStateName(newStateName, showWarnings));
       },
+      getStateContentMemento: function(stateName) {
+        return getStatePropertyMemento(stateName, 'content');
+      },
+      saveStateContent: function(stateName, newContent) {
```

Comment:
Wondering if this should be 'setStateConent' instead. 'save' sort of implies it is retaining an existing state, whereas this is overriding and retaining. 'set' seems to fit better for this and all other new 'save*' functions (same with ``saveStateProperty``)